pages.txt

```diff
@@ -3,70 +3,71 @@ Idobata makes software development fun
 
 By developers for developers
 Lightweight, familiar UI inspired by IRC
-Integration with the tools that you use
+Integrates with the tools that you use
 Sign Up
 
-Free for good
+Free forever
 
+Why?
+There are tons of reasons to use Idobata.
 
-WHY?
-WHY IDOBATA
-We made Idobata to solve developer's worries about team development.
+Idobata solves a lot of common problems with developer chat tools. 
 
-It's painful to maintain server for IRC...
+It's painful to maintain server for IRC.
 
-I wish I could @username style mention with type-ahead...
+I wish I could autocomplete my @username-style mentions. 
 
-I want easy and adequate integration with GitHub, CI and so on...
+I want simple integration with GitHub, CI, etc... 
 
-How to see transversely all messages which I am interested in?
+I'd like to see every message thread I am following in at a glance.
 
-Developer's chat should be more programmable...
+Developers' chat should be more programmable.
 
-ORGANIZATION
-ONE ACCOUNT, MULTIPLE TEAMS
```

Comment:
The CSS for this page make this entire line uppercased. I'd like to leave it if possible because it's a part of the visual design. May I do it?
(Note: I don't understand how important this change is.)